pkg/asset/installconfig/ovirt/validaton.go

```diff
@@ -40,9 +40,46 @@ func Validate(ic *types.InstallConfig) error {
 			field.Invalid(ovirtPlatformPath.Child("vnicProfileID"), ic.Ovirt.VNICProfileID, err.Error()))
 	}
 
+	if ic.ControlPlane != nil && ic.ControlPlane.Platform.Ovirt != nil {
+		allErrs = append(
+			allErrs,
+			validateMachinePool(
+				con,
+				field.NewPath("controlPlane", "platform", "ovirt"),
+				ic.ControlPlane.Platform.Ovirt)...)
```

Comment:
hmm.. can single line this?

Reply:
done.
The reason I do those if for lines longer than 120.  I see no guidelines on the project for this and I think its much easier to read it like that.